src/cards/ScenarioNode/ScenarioNode.js

```diff
@@ -35,9 +35,11 @@ export const ScenarioNode = ({
   deleteScenario,
   checkScenarioNameValue,
   canRenameScenario,
+  canUpdateScenario,
```

Comment:
My idea was to keep both props (and corresponding functions) till a refactoring of all update cases even if they are identical indeed. I don't fully understand the suggestion : you think we can make a intermediate refactoring of `EditableLabel` to accept both props ? Is it's so, I'm not sure about it because it means a new version of the component for some weeks :thinking:

Reply:
What I had in mind was just to simplify this change on `ScenarioNode` by removing one of the two props `canRenameScenario` and `canUpdateScenario`, but it can wait for the refactoring that is already planned :smile:

Reply:
I prefer to keep it simple and refactor everything together :wink: